ui/src/app/views/workspace-list/component.html

```diff
@@ -1,9 +1,10 @@
 <div class="area-block">
   <div class="table-header">
     <h3 class="table-header-descriptor">Your Workspaces</h3>
-    <button class="btn btn-secondary" (click)="addWorkspace()">
+    <button class="btn btn-secondary tooltip tooltip-lg tooltip-top-left" (click)="addWorkspace()" [disabled]="!billingProjectInitialized">
       <clr-icon shape="plus"></clr-icon>
       New Workspace
+      <span *ngIf="!billingProjectInitialized" class="tooltip-content">Your billing project is currently being created, please wait.</span>
```

Comment:
Nit: Line-length? Just a question. Not sure if we have a standard for HTML templates.

Reply:
We haven't yet established a convention. I don't have any strong feelings on it though? Probably just go with the same 100 char? @johnstill do you have any feelings?

Reply:
My only real tendency is that if we're going to wrap a tag, wrap it at every attribute - so, multiple attrs on a line _or_ multiple lines for a tag, but not both.  Both gets soupy and hard to read.  I would break this here at the content.

```
<tag>
  Text content
</tag>
```

The extra whitespace around the text node will be stripped on render (generally), so we can use it for our own formatting.

Reply:
I 100% agree with the idea of 
```
<tag>
  Text content
</tag>
```
that is also my preferred formatting there. And then for text content/tags, I am happy going with a 100 char limit for lines there, I think that keeps some consistency between TS and HTML.